Set better default value for RFC_STRICT servers that do not show ".." in LIST
diff -urN mc-4.7.5.5.orig/lib/vfs/mc-vfs/parse_ls_vga.c mc-4.7.5.5/lib/vfs/mc-vfs/parse_ls_vga.c
--- mc-4.7.5.5.orig/lib/vfs/mc-vfs/parse_ls_vga.c	2011-09-28 12:49:09.000000000 -0400
+++ mc-4.7.5.5/lib/vfs/mc-vfs/parse_ls_vga.c	2011-10-19 13:14:30.000000000 -0400
@@ -648,7 +648,7 @@
 void
 vfs_parse_ls_lga_init (void)
 {
-    vfs_parce_ls_final_num_spaces = 0;
+    vfs_parce_ls_final_num_spaces = 1;
 }
 
 /* --------------------------------------------------------------------------------------------- */
